frontend/src/pages/CreateNFTPage.jsx

```diff
@@ -14,12 +16,68 @@ const CreateNFT = ({ wallet }) => {
         file: null,
     })
 
+    const [dragging, setDragging] = useState(false); 
+    const fileInputRef = useRef(null);
+    const textareaRef = useRef(null);
+
+    const adjustTextareaHeight = () => {
```

Comment:
Duplicate assignment statement
## deepsource : Consecutively reassigning to the same variable or property is a code smell and should be avoided.
```html
Bad Practice

a[1] = 'something'
a[1] = 'some other thing'

Recommended

a[1] = 'something'
if (condition()) {
  a[1] = 'some other thing'
}


```